### DISCUSSION TITLE
Question & Suggestion(s): Inverse Dimensions and Inverse Players

### DISCUSSION THREAD

**Discussion by Onemario1234** [iPortalTeam/ImmersivePortalsMod#514] [Ideas]
I am aware of the planned overhaul for Dimension Stacking, and the plan to add the ability to invert Dimensions in a stack, and I am excited for this! However, I do have a question regarding inverse Dimensions, and after this question, I have a suggestion.

EDIT: The following question has been answered.
As for the question, how are Inverse Dimensions planned to work? Will you enter an inverse Dimension and suddenly flip to the Dimension's orientation? Or will you enter an Inverse Dimension and start roaming said Dimension upside-down?

The suggestion is assuming it's the latter. If it is the latter, I suggest adding an option to toggle. When off, you enter the Dimension upside-down. When on, you flip to the correct orientation upon entering.

**Comment by Ookidablobida** [comment]
Maybe you could even set a y level to flip upright at, or somehow detect when the player has dug through the ground. I would probably suggest the first one, because if they add the gravity as an nbt tag or something, we could find some way to do the second one without making the creators' lives harder.

**Reply by Onemario1234** [reply]
That's what I was thinking too, though perhaps I should've explained as much.

Switching the player's gravity (as opposed to flipping the entire world) would be far easier. Heck, it's already been done in a 1.12.2 mod, Up And Down And All Around.
